sync/class.jetpack-sync-defaults.php

```diff
@@ -104,6 +104,8 @@ class Jetpack_Sync_Defaults {
 		'users_can_register',
 		'active_plugins',
 		'uninstall_plugins',
+		'advanced_seo_front_page_description',
+		'advanced_seo_title_formats',
 	);
```

Comment:
Maybe we can add a comment so when someone searches for DESCRIPTION_META_KEY in the code base that they can find it.